core/src/main/java/io/parsingdata/metal/data/selection/ByToken.java

```diff
@@ -92,12 +92,12 @@ public static ParseItem get(final ParseGraph graph, final Token definition) {
 
     private static ImmutableList<ParseItem> getAllRootsRecursive(final ParseItem item, final ParseGraph parent, final Token definition) {
```

Comment:
Add some Javadoc on the `public` method to explain the ordering of the resulting list.